src/transformers/training_args.py

```diff
@@ -227,6 +228,9 @@ class TrainingArguments:
         fp16_full_eval (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to use full float16 evaluation instead of 32-bit. This will be faster and save memory but can harm
             metric values.
+        tf32 (:obj:`bool`, `optional`, defaults to :obj:`None`):
```

Comment:
```suggestion
        tf32 (:obj:`bool`, `optional`):
```
We never put the "defaults to `None`" as it's implied by the "optional".